### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by Eilon**
@Tratcher

### CODE REVIEW

**Comment by Eilon** [comment]
Merged, but please have a look to ensure the metadata is good.

File: src/Microsoft.AspNetCore.Owin/project.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "version": "1.0.0-*",
-  "description": "ASP.NET 5 component for running OWIN middleware.",
+  "description": "ASP.NET Core component for running OWIN middleware in an ASP.NET Core application.",
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Owin/project.json]: And for running ASP.NET Core middleware in OWIN based applications.

File: src/Microsoft.Net.Http.Headers/project.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "version": "1.0.0-*",
-  "description": "ASP.NET 5 HTTP header implementations.",
+  "description": "ASP.NET Core HTTP header parser implementations.",
```
Comment (Tratcher) [inline src/Microsoft.Net.Http.Headers/project.json]: This package is considered independent of ASP.NET.

File: src/Microsoft.AspNetCore.Owin/project.json (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.Owin/project.json]: Oh nice, I didn't even realize that. Will fix.

File: src/Microsoft.Net.Http.Headers/project.json (hunk 2 above)
Comment (Eilon) [inline src/Microsoft.Net.Http.Headers/project.json]: Will fix.

**Comment by Eilon** [comment]
https://github.com/aspnet/HttpAbstractions/pull/599 has the follow-up fixes.

**Pull request merged**